spec/features/idv/usps_otp_prefilling_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+feature 'idv usps otp prefilling', :idv_job do
```

Comment:
Does this require a feature spec? All we're testing here is that the view receives the code when the feature flag is on, and that it doesn't receive the code when the flag is off. Can't this be tested with a controller and view spec?

Reply:
Probably not for the positive case, but I do think we'll want to test the negative case since we do get in trouble if the code starts showing up when it isn't supposed to.